formulas/functions/math.py

```diff
@@ -266,7 +268,10 @@ def xroman(num, form=0):
 FUNCTIONS['ROMAN'] = wrap_ufunc(xroman, input_parser=lambda *a: a)
 
 
-def xround(x, d, func=round):
+def round_up(x):
+    return float(Decimal(x).quantize(0, rounding=ROUND_HALF_UP))
+
+def xround(x, d=0, func=round_up):
```

Comment:
can you write an example/test case where the previous function was failing?

Reply:
Sorry for the delayed response. I will try to come up with a test case for this. Thanks for adding all the new functions is 0.4 I can retest with 0.4 and see if the round issues have gone away too.

Reply:
Thanks!

Reply:
I created a new PR with just this change. It is needed to match excel round formula. Added a test that shows the issue.